ecosystem/sep-0024.md

```diff
@@ -207,7 +208,8 @@ Request parameters:
 
 Name | Type | Description
 -----|------|------------
-`asset_code` | string | Code of the asset the user wants to withdraw. This must match the asset code issued by the anchor. Ex if a user withdraws MyBTC tokens and receives BTC, the `asset_code` must be MyBTC.
+`asset_code` | string | Code of the asset the user wants to withdraw. This must match the asset code issued by the anchor. Ex if a user withdraws MyBTC tokens and receives native BTC, the `asset_code` must be MyBTC.
```

Comment:
> This must match the asset code issued by the anchor.

This should be removed right? The anchor can back an asset it didn't issue

Reply:
yep